roles/devscripts/vars/main.yml

```diff
@@ -37,16 +37,12 @@ cifmw_devscripts_config_defaults:
   cluster_name: "ocp"
   base_domain: "openstack.lab"
   ntp_servers: "clock.corp.redhat.com"
-  ip_stack: "v4"
-  network_type: "OVNKubernetes"
   provisioning_network_profile: "Managed"
   provisioning_network: "172.22.0.0/24"
   cluster_subnet_v4: "192.168.16.0/20"
   cluster_host_prefix_v4: "22"
   service_subnet_v4: "172.30.0.0/16"
   external_subnet_v4: "192.168.111.0/24"
   num_masters: 3
-  master_memory: 32768
-  master_disk: 80
-  master_vcpu: 10
   num_workers: 0
+  ip_stack: v4  # TODO(Harald) Check if we can get rid of it - see 133_host_network.yml
```

Comment:
So, the condition to select the `ansible_default_ipv6.interface` and then set up firewall rules in the ipv4 table to do masquerading on traffic going out that interface does not make any sense to me.

But is'nt this also used to set the ENV vars for the dev-scripts installer? See: `roles/devscripts/templates/conf_ciuser.j2` there is a snippet converting these to "shell vars" used by devscripts.

```
{% for item in cifmw_devscripts_config %}
export {{ item.upper() }}="{{ cifmw_devscripts_config[item] }}"
{% endfor %}
```

I think `NETWORK_TYPE` and `IP_STACK` is something we need to define for dev-scripts.

Reply:
IP_STACK and NETWORK_TYPE have "good" default values:
https://github.com/openshift-metal3/dev-scripts/blob/3c655f4653f575009b6c92796f726d75e89027aa/config_example.sh#L220-L227

https://github.com/openshift-metal3/dev-scripts/blob/3c655f4653f575009b6c92796f726d75e89027aa/config_example.sh#L283-L294

I propose we keep that one in - I tried to loop on the interfaces but faced some weird failure. "it's working", we may work with some follow-up.. ?

Reply:
Yeah - let's just keep them in. And we can try to clean up later.
Marking resolved.